packages/roosterjs-react-command-bar/lib/utils/OutOfBoxCommands.ts

```diff
@@ -0,0 +1,130 @@
+// TODO-1 import { ICommandBarItemProps } from '@uifabric/experiments/lib/CommandBar';
+import { IContextualMenuItem } from 'office-ui-fabric-react/lib/ContextualMenu';
+import { IIconProps } from 'office-ui-fabric-react/lib/Icon';
+import {
+    clearFormat,
+    removeLink,
+    setAlignment,
+    setIndentation,
+    toggleBold,
+    toggleBullet,
+    toggleItalic,
+    toggleNumbering,
+    toggleStrikethrough,
+    toggleSubscript,
+    toggleSuperscript,
+    toggleUnderline,
+} from 'roosterjs-editor-api';
+import { Editor } from 'roosterjs-editor-core';
+import { Alignment, FormatState, Indentation } from 'roosterjs-editor-types';
+
+function getIconProps(name: string): IIconProps {
+    return { className: `$rooster-command-bar-icon ms-Icon ms-Icon--${name}` };
+}
+
+export const OOB_COMMAND_BAR_ITEMS: IContextualMenuItem[] = [
+    {
```

Comment:
Do we get tooltip/aria-labelf or free with this or do we need to set it?

Reply:
no we don't, just need to pass in buttonNames for localized name and title. Since the name value is inside the native button element, aria label isn't needed